Core/src/org/sleuthkit/autopsy/logicalimager/dsp/AddLogicalImageTask.java

```diff
@@ -454,12 +507,19 @@ private void addExtractedFiles(File src, Path resultsPath, List<Content> newData
                         Long.parseLong(atime),
                         Long.parseLong(mtime),
                         localFilesDataSource);
-
+                    String key = String.format("%s\t%s", ruleSetName, ruleName);
+                    List<Long> value = new ArrayList<>();
```

Comment:
This could also probably be done with using the "value" list

Reply:
I can't here. I am appending one fileId to the value(s) of a key (ruleSetName + ruleName).